.github/stale.yml

```diff
@@ -1,25 +1,34 @@
 # Number of days of inactivity before an issue becomes stale
 daysUntilStale: 30
+
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 14
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels:
+  - "additional information requested"
+
 # Issues with these labels will never be considered stale
 exemptLabels:
   - backlog
   - epic
   - pinned
   - security
   - triage
```

Comment:
Why keep these in here if the only thing we will take are issues that need more info?

Reply:
What about the case where it has both labels?

Reply:
If it needs more info to be actionable, I'd be okay with it being marked as stale even if it has any of those other labels.